doc/UsersGuide/source/baroclinic_wave.rst

```diff
@@ -0,0 +1,103 @@
+.. role:: raw-html(raw)
+    :format: html
+
+.. _baroclinic-wave:
+
+*****************************
+Baroclinic Instability Case
+*****************************
+
+The UFS WM baroclinic wave case adapts the test outlined in :cite:t:`Jablonowski&Williamson2006` (2006). This test is designed to evaluate the accuracy of various atmospheric models in simulating a baroclinic wave, which commonly forms in the Northern Hemisphere and influences weather patterns. This test aims to assess how well "dry dynamical cores," the foundational components of weather and climate models that handle air movement and temperature changes, perform in idealized conditions. 
+
+The simulation sets the model's atmosphere to an initial steady state, designed to be a simple, realistic representation of atmospheric conditions using the adiabatic (no heat exchange) and inviscid (no friction) primitive equations. The test first checks whether each model can maintain this steady, zonal (west-to-east) state without developing any unintended changes. After verifying this, the next step is to introduce a small disturbance, or perturbation, which triggers the growth of a baroclinic wave. The wave then evolves over several simulated days, allowing the researchers to observe how accurately different models handle the wave's development and movement.
+
+This test provides a standard way to assess how different atmospheric models handle the development of baroclinic waves. The results help identify which models are more accurate and can serve as benchmarks for model improvement, ultimately contributing to better simulations of atmospheric behavior in weather and climate predictions.
+
+In the UFS WM, the idealized baroclinic wave test case is an atmosphere-only, :term:`dycore`-only forecast run at C192 resolution with 127 vertical levels. It uses default values from the WM's ``export_fv3`` function, along with default values for a tiled grid namelist (from ``export_tiled``) and for the `Unified Gravity Wave Physics version 1 <https://dtcenter.ucar.edu/GMTB/v7.0.0/sci_doc/ugwpv1_gsldrag.html>`_ (from ``export_ugwpv1``). These initial values are all set based on values from `default_vars.sh <https://github.com/ufs-community/ufs-weather-model/blob/develop/tests/default_vars.sh>`_. 
```

Comment:
I would remove and for the `Unified Gravity Wave Physics version 1 <https://dtcenter.ucar.edu/GMTB/v7.0.0/sci_doc/ugwpv1_gsldrag.html>`_ (from ``export_ugwpv1``) since only dynamics are used in this test (i.e., no physics). Input namelist values are hard-coded and do not get assigned for this case based on the export_ugwpv1 section of default_vars.sh.